torch/_deploy.py

```diff
@@ -0,0 +1,74 @@
+import io
+import torch
+import importlib
+from torch.package._custom_import_pickler import create_custom_import_pickler
+from torch.package.importer import _UnpicklerWrapper
+from torch.package import PackageImporter
+from torch.serialization import _maybe_decode_ascii
+from typing import Callable
+from types import ModuleType
+
+def _save_storages(importer, obj):
```

Comment:
is the idea here that you could either use torch.package to do the save, or not use it?  I don't really understand torch.package especially what it means to create a custom import pickler without using torch.package's imporer

Reply:
a custom importer will resolve code objects from within the package. The importer here gets set if the MovableObject we are working with was produced from a package.